pyrightconfig.stricter.json

```diff
@@ -94,6 +94,7 @@
         "stubs/setuptools/setuptools",
         "stubs/shapely",
         "stubs/tensorflow",
+        "stubs/TgCrypto",
```

Comment:
It seems like the type annotations for this package are complete, so this exclusion isn't necessary; we can just use the stricter pyright settings on this package:

```suggestion
```